downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc

```diff
@@ -0,0 +1,108 @@
+[id="ref-single-controller-hub-eda-with-managed-db"]
+
+=  Single controller, single automation hub, and single event-driven ansible controller node with external (installer managed ) database
+
+[role="_abstract"]
+Use this example to populate the inventory file to deploy single instances of {ControllerName} and {HubName} with an external (installer managed) database.
+
+[IMPORTANT]
+====
+This scenario requires a minimum of {ControllerName} 2.4 for successful deployment of {EDAcontroller}.
+====
```

Comment:
Is there any way to rewrite this so there aren't two Important messages in a row? I wonder if the first notice couldn't be added to the Intro?